run_bot_common.sh

```
@@ -0,0 +1,116 @@
+}
+
+# Process JSON configuration to add UUID and streaming params if missing
+process_config() {
+    local config_json="$1"
+    local bot_uuid
+    bot_uuid=$(generate_uuid)
+    print_info "${ICON_BOT} Generated bot session ID: ${bot_uuid:0:8}..."
+
+    if command -v jq &> /dev/null; then
+        # Build jq command to update bot_uuid and optionally streaming params
+        local jq_cmd='.bot_uuid = $bot_uuid'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)